### PR TITLE
feat: add edit distance evaluator

### PR DESCRIPTION
**Pull request by bhavsarpratik**
Add edit distance evaluator

### CODE REVIEW

File: evaluators/text/EditDistanceEvaluator/tests/test_editdistanceevaluator.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from .. import EditDistanceEvaluator
+
+
+@pytest.mark.parametrize('actual, desired, distance',
```
Comment (JoanFM) [inline evaluators/text/EditDistanceEvaluator/tests/test_editdistanceevaluator.py:6]: What happens when mixed strings of different lengths not empty and also capital letters?

**Review by JoanFM — CHANGES_REQUESTED** [review]
Just some edge cases

File: evaluators/text/EditDistanceEvaluator/tests/test_editdistanceevaluator.py (hunk 1 above)
Comment (JoanFM) [reply evaluators/text/EditDistanceEvaluator/tests/test_editdistanceevaluator.py:6]: What if sentences with punctuation marks?

**Review by JoanFM — APPROVED** [review]
